nvflare/private/fed/app/simulator/simulator_worker.py

```diff
@@ -64,6 +64,7 @@ def create_client_runner(self, client):
 
         client_app_runner = SimulatorClientAppRunner()
         client_runner = client_app_runner.create_client_runner(app_client_root, args, args.config_folder, client, False)
+        client_app_runner.sync_up_parents_process(client)
```

Comment:
Why do we need this line?

Reply:
Use the consistent approach as the server to sync up data with the parent process.